Loop/gallery.ckcomplication/Base.lproj/ckcomplication.strings

```diff
@@ -1,16 +0,0 @@
-/* 
```

Comment:
I'm having problems with it generating the correct files at the moment. I did regenerate them before I added this PR but it clearly didn't add in the other file. I will look into this now and resolve it.